Added AnVIL Office Hours for January 2023 #2473 (#2481)
anvilproject/anvil-portal: content/events/anvil2023-january-office-hours.md

---
author: "AnVIL"
conference: "AnVIL 2023"
description: "Helping users find solutions to their AnVIL questions."
eventType: "Office Hours"
featured: true
location: "Virtual"
sessions: [{sessionStart: "26 Jan 2023 11:00 AM"}]
timezone: "America/New_York"

> title: "AnVIL Office Hours - February 2023"

Reviewer: Hi @jpaten would you mind changing the month in the title from `February` to `January`?

---

<event-hero></event-hero>

### Description

> These Office Hours are a new opportunity for you to get your questions about working on AnVIL answered in person –

Reviewer: ```suggestion
These Office Hours are an opportunity for you to get your questions about working on AnVIL answered in person –
```

Reviewer: @jpaten I think the `Description` text in the ticket has been updated since previous Office Hours... would you mind updating as per the ticket text?

whether you’re trying to set up a billing account, launch Galaxy or RStudio, looking for methods and featured
workspaces, and more.

### Audience
Members of the AnVIL team will be available to help AnVIL users including PIs, analysts, and data submitters get unstuck, troubleshoot issues, and discover online resources that provide further information.

### Prerequisites
Please post your questions to the [AnVIL Support Forum](https://help.anvilproject.org/t/anvil-office-hours-26jan2023-11-am-et/146) prior to Office Hours so we can bring the team representatives best suited to address your question.

> ## Event Details

Reviewer: Hi @jpaten for consistency, for the registration and contact information section, take a look at e.g. http://localhost:8000/events/anvil2022-april-office-hours as an example for the section title and following text.

e.g.

Remove ## Event Details and

```
### Registration
Post your ....

### Contact
Please contact ...
```
